mergin/client.py

```diff
@@ -10,9 +10,14 @@
 import math
 import hashlib
 from datetime import datetime, timezone
+import concurrent.futures
 
 this_dir = os.path.dirname(os.path.realpath(__file__))
-CHUNK_SIZE = 10 * 1024 * 1024
+
+CHUNK_SIZE = 100 * 1024 * 1024
```

Comment:
Is this intentionally changed to 100mb?

Reply:
Yes. This value should be kind of trade-off between increasing number of requests and admitting some risk of failure during download and subsequent retry.